src/ocean_component.cpp

```diff
@@ -81,6 +81,18 @@ void OceanComponent::init( Core* coreptr ) {
     core->registerCapability( D_TU, getComponentName() );
     core->registerCapability( D_TWI, getComponentName() );
     core->registerCapability( D_TID, getComponentName() );
+    core->registerCapability( D_PH_HL, getComponentName() );
+    core->registerCapability( D_PH_LL, getComponentName() );
+    core->registerCapability( D_ATM_OCEAN_FLUX_HL, getComponentName() );
+    core->registerCapability( D_ATM_OCEAN_FLUX_LL, getComponentName() );
+    core->registerCapability( D_PCO2_HL, getComponentName() );
+    core->registerCapability( D_PCO2_LL, getComponentName() );
+    core->registerCapability( D_DIC_HL, getComponentName() );
+    core->registerCapability( D_DIC_LL, getComponentName() );
+    core->registerCapability( D_TEMP_HL, getComponentName() );
+    core->registerCapability( D_TEMP_LL, getComponentName() );
+    core->registerCapability( D_CO3_HL, getComponentName() );
+    core->registerCapability( D_CO3_LL, getComponentName() );
```

Comment:
@bpbond  I assume that we want to return ph, the ocean fluxes, pco2, dic, temp,  and co3 correct? For the high and low latitude, we only want to return the surface level values, we don't want to sum over the intermediate and deep occeans correct?

Reply:
That seems like a good place to start. Yes, just surface values; there's no chemistry in the intermediate and deep boxes.